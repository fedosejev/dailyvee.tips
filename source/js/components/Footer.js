import React from 'react';

const Footer = () => (
  <div className="footer">
    Created by <a href="https://artemij.com" target="_blank" rel="noopener noreferrer">Artemij Fedosejev</a>.
  </div>
);

export default Footer;
