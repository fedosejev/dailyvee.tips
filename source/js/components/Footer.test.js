import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';

describe('Footer', () => {
  test('SHOULD render correctly', () => {
    const tree = renderer.create(
      <Footer />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
