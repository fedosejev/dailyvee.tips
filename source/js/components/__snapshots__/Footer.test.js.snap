// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer SHOULD render correctly 1`] = `
<div
  className="footer"
>
  Created by 
  <a
    href="https://artemij.com"
    rel="noopener noreferrer"
    target="_blank"
  >
    Artemij Fedosejev
  </a>
  .
</div>
`;
